src/prefect/cli/server.py

```diff
@@ -68,9 +68,23 @@ def server():
         is_flag=True,
         hidden=True,
     ),
+    click.option(
+        "--external-postgres",
+        "-ep",
+        help="Pass this flag to use an external postgres instance",
+        is_flag=True,
+        hidden=True,
+    ),
+    click.option(
+        "--postgres-url",
+        help="Postgres url to use",
```

Comment:
"Custom postgres connection url to use. This will remove the postgres service from the docker-compose setup. Expected format is ....?"